app/javascript/components/public_health/CustomExport.js

```diff
@@ -253,6 +253,17 @@ class CustomExport extends React.Component {
                               {['search', 'select', 'number'].includes(filter.filterOption?.type) && (
                                 <span>{filter.value === '' ? '<blank>' : filter.value}</span>
                               )}
+                              {filter.filterOption?.type === 'multi' && (
+                                <div style={{ display: 'inline-grid' }}>
+                                  {filter.value?.map((elem, i) => {
```

Comment:
Seeing this error after I apply blank filters like this and open the custom export modal:

![Screen Shot 2021-08-10 at 1 24 18 PM](https://user-images.githubusercontent.com/9956561/128906179-335f924f-f22b-4128-aee8-7f56b718e27f.png)


![Screen Shot 2021-08-10 at 1 23 04 PM](https://user-images.githubusercontent.com/9956561/128906018-3e2f575f-7b3b-4165-a8e0-f618e568cab5.png)

Reply:
I think this has been fixed with your most recent changes!